samples/SkijaInjectSample/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+    id 'org.jetbrains.kotlin.jvm' version '1.4.21'
```

Comment:
Why this change?

Reply:
Ok, reverted to 1.3.72. I think we don't need features of 1.4